views/movies/movie-details.hbs

```diff
@@ -0,0 +1,17 @@
+<h2>Movie Details</h2>
+
+<h3>Title: {{movie.title}}  </h3>
+<h4>Genre: {{movie.genre}} </h4>
+<h4>Plot: {{movie.plot}} </h4>
+
+{{#each celebrities }}
+
+<h2>{{name}}</h2>
+<p> {{occupation}} </p>
+<p> {{catchPhrase}} </p>
+  
+{{/each}}
+
```

Comment:
You are missing an anchor tag `<a>` that goes to the `edit-movie` view. Maybe it could be here on the movies details